easybuild/tools/config.py

```diff
@@ -843,6 +845,33 @@ def log_path(ec=None):
     return log_file_format(return_directory=True, ec=ec, date=date, timestamp=timestamp)
 
 
+def error_log_path(ec=None):
+    """
+    Return the default error log path
+
+    This is a path where file from the build_log_path can be stored permanently
+    :param ec: dict-like value that provides values for %(name)s and %(version)s template values
+    """
+    error_log_path = ConfigurationVariables()['errorlogpath']
```

Comment:
Not a good idea to have a variable with the same name as a function. Maybe rename the function to `get_....` or `default_...`?